### ISSUE TITLE
Flaticon_circle.png gets REPLACED to 0 filesize

### ISSUE THREAD

**Issue by TripleNothing** [plan-player-analytics/Plan#1194]
### Describe the issue
When the web folder's root items are created, all seems fine EXCEPT the img/Flaticon_circle.png file. A file is created, but of 0 filesize. If I manually replace it, it will successfully display on the webpage, but after a minute or so, upon its next update, it gets replaced by the original 0 byte file.

### Server information
Ubuntu 18.04 LTS
BungeeCord Build 1420 (Most recent at this moment)
PaperSpigot 1.14.4 Build 214 (Most recent at this moment)
Player Analytics 5.0 Build 224 (Most recent at this moment)
  - Plan was set up to use a local MySQL and its internal web directory, but hosted locally by Apache.

### Exceptions & Other Logs
Nothing logged by this that I found, but an example reference. 
(Edited/clipped to shorten)

> minecraft@group:.../BungeeCord/plugins/Plan/web/img$ ls -l
> total 0
> -rw-rw-r-- 1 minecraft minecraft 0 Oct 15 16:41 Flaticon_circle.png
> minecraft@group:.../BungeeCord/plugins/Plan/web/img$ rm Flaticon_circle.png
> minecraft@group:.../BungeeCord/plugins/Plan/web/img$ wget https://github.../Flaticon_circle.png
> Length: 18687 (18K) [image/png]
> Saving to: ‘Flaticon_circle.png’
> Flaticon_circle.png 100%[===========================>]  18.25K  --.-KB/s    in 0.03s
> 2019-10-15 16:48:19 (654 KB/s) - ‘Flaticon_circle.png’ saved [18687/18687]
> minecraft@group:.../BungeeCord/plugins/Plan/web/img$ ls -l
> total 20
> -rw-rw-r-- 1 minecraft minecraft 18687 Oct 15 16:48 Flaticon_circle.png
> minecraft@group:.../BungeeCord/plugins/Plan/web/img$ ls -l
> total 0
> -rw-rw-r-- 1 minecraft minecraft 0 Oct 15 16:51 Flaticon_circle.png

**Comment by TripleNothing** [comment]
Well, after a few tests, I discovered the valid file is INITIALLY present. It's just as times goes on, perhaps as it runs each update/alter, it then gets changed to 0 filesize.

Flaticon_circle.png | 2019-10-17 07:26 | 18K
Flaticon_circle.png | 2019-10-17 07:33 | 0

I did hop on the server for a moment and off to see if the timing on this matched with the creation of the 'server' directory, since it doesn't even exist upon the initial setup with the rest, but the file zero'ed out before the 'server' directory even exists in the first place as well.

Hope the timing helps reference what may be doing this. ^_^

**Comment by Rsl1122** [comment]
I was not able to reproduce this issue on Windows, but this might be a Linux specific bug.
A change was made to try to fix this anyhow.

**Comment by TripleNothing** [comment]
I did rebuild the server running this for a clean slate. It runs the same, and changes it to 0 filesize after some time (a minute or so), but is perfect to start.

Once the file exists, is there anything beyond an initial "if exists" or such to create the file? Or is it updated/replaced every page build? Sorry for my reference.

I did define it as a read-only to hold it for now, but am more than willing to test anything new. I'm the project type. ;-) (But slid more towards web & less plugins/addons over the years. :-/ )
